### DISCUSSION TITLE
calculating electricity prices

### DISCUSSION THREAD

**Discussion by ssattler** [NREL/ReEDS-2.0#74] [Q&A: Analysis] [answered]
I have run a number of state RPS scenarios, we are ultimately looking for state-level retail electricity prices. I'm seeing energy prices, the bulk system electricity price, national energy price, national average electricity cost, and other prices/costs for regional, capacity operating reserve, etc. Can you help?

**Comment by wesleyjcole** [comment, marked as answer]
Retail rate outputs are not included in the 2020 model version.  They will be included in the 2021 model version, which will hopefully be released within the next month.

**Reply by ssattler** [reply]
Is there a way to approximate retail rates given the outputs from the 2020 model version?

**Reply by wesleyjcole** [reply]
Not without a lot of work.  ReEDS only captures the costs of bulk power system assets.  It does not include administration, distribution, and other costs.
